dub.sdl

```diff
@@ -6,7 +6,7 @@ homepage "http://tour.dlang.org"
 license "Boost"
 dependency "vibe-d" version="~>0.7.31-rc.3"
```

Comment:
Can you also update vibe.d to 0.7.31, i.e. non-rc? (I'm not sure if vibe.d 0.8 is ready fo production, Sonke mentioned at dconf that there were some performance issues that he needed to investigate.)